headapps/spa-starters/.gitignore

```diff
@@ -0,0 +1,5 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+# dependencies
+/node_modules
+
+.netlify
```

Comment:
speaking of this, let's include netlify.toml out of the box.
It's a starter kit - so we can provide everything clients can start with.